market-service/src/main/java/rs/edu/raf/banka1/repositories/OptionsRepository.java

```diff
@@ -10,4 +13,10 @@
 @Repository
 public interface OptionsRepository extends JpaRepository<OptionsModel, Long> {
     Optional<List<OptionsModel>> findByTicker(String ticker);
+
+    @Transactional
```

Comment:
da li ovde treba transactional posto se sve izvrsava u metodi koja je transactional?

Reply:
Puca mi ovaj exception kad nemam u repository: Exception in thread "Thread-12" org.springframework.dao.InvalidDataAccessApiUsageException: Executing an update/delete query
Tako da sam ga stavila i ovde